src/python/pants/backend/docker/util_rules/docker_build_context.py

```diff
@@ -192,7 +193,8 @@ def _get_stages_and_tags(
             if idx == 0:
                 # Expose the first (stage0) FROM directive as the "baseimage".
                 tags_values["baseimage"] = tag
```

Comment:
I think this `if idx == 0` should also be included in the new `if tag:` too, or else it might set an empty tag.

Reply:
Done. Thanks.